recipe/meta.yaml

```diff
@@ -54,16 +53,19 @@ outputs:
       - bin/curl-config          # [unix]
       - Library/bin/libcurl.dll  # [win]
       - Library/include/curl     # [win]
-      - Library/lib/libcurl*     # [win]
+      - Library/lib/libcurl.lib  # [win]
+      - Library/lib/libcurl.exp  # [win]
     test:
       commands:
         - curl-config --features       # [not win]
         - curl-config --protocols      # [not win]
         - test -f ${PREFIX}/lib/libcurl${SHLIB_EXT}  # [not win]
         - test ! -f ${PREFIX}/lib/libcurl.a          # [not win]
+        - if exist %LIBRARY_BIN%\curl.exe exit 1     # [win]
```

Comment:
Do we still want this test here? Do we want to `curl.exe` in the `libcurl` package?

FWIW appears we are also testing it in the `curl` package below.

Reply:
Yes, we want the test to ensure `curl.exe` is not in `libcurl` package

Reply:
Got it. For some reason I missed the `exit 1`. Makes sense.